runtime/src/lib.rs

```diff
@@ -631,6 +691,12 @@ construct_runtime!(
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 11,
 
 		// Collator support. The order of these 4 are important and shall not change.
+		//
+		// TODO: Do we want to add these yet?
```

Comment:
I don't see any reason for not adding it. It won't add any complexity in terms of development or testing.